[8.7][CSPM] Adds CSPM overview and getting started pages to Cloud Security section (#3014)
elastic/security-docs: docs/cloud-native-security/cspm.asciidoc

[[cspm]]
= Cloud security posture management

The Cloud Security Posture Management (CSPM) integration discovers and evaluates the services in your cloud environment, like storage, compute, IAM, and more, against configuration security guidelines defined by the https://www.cisecurity.org/[Center for Internet Security] (CIS) to help you identify and remediate risks that could potentially undermine the confidentiality, integrity, and availability of your cloud data.

> This integration currently supports Amazon Web Services (AWS). Support for other major cloud service providers is coming soon. For a step-by-step getting started guide, refer to <<cspm-get-started,Get started with CSPM>>.

Reviewer: Just putting it out there: we typically avoid forward-looking statements that seem to promise future features or support, for best practices and maybe also legal reasons. There are also several similar statements in the FAQs, maybe elsewhere.

.Requirements
[sidebar]
--

> * The CSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].

Reviewer: ```suggestion
* The CSPM integration is available to all Elastic Cloud users. On-premises deployments require an https://www.elastic.co/pricing[Enterprise subscription].
```
